intellij-toml/src/main/kotlin/org/toml/ide/formatter/settings/TomlLanguageCodeStyleSettingsProvider.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml.ide.formatter.settings
+
+import com.intellij.application.options.IndentOptionsEditor
+import com.intellij.application.options.SmartIndentOptionsEditor
+import com.intellij.lang.Language
+import com.intellij.psi.codeStyle.CodeStyleSettingsCustomizable
+import com.intellij.psi.codeStyle.CommonCodeStyleSettings
+import com.intellij.psi.codeStyle.LanguageCodeStyleSettingsProvider
+import org.toml.lang.TomlLanguage
+
+class TomlLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider() {
+    override fun getLanguage(): Language = TomlLanguage
+    override fun getCodeSample(settingsType: SettingsType): String =
+        when (settingsType) {
+            SettingsType.SPACING_SETTINGS -> SPACING_SAMPLE
+            else -> ""
+        }
+
+    override fun customizeSettings(consumer: CodeStyleSettingsCustomizable, settingsType: SettingsType) {
+        @Suppress("NON_EXHAUSTIVE_WHEN")
+        when (settingsType) {
+            SettingsType.SPACING_SETTINGS -> {
+                consumer.showStandardOptions(CodeStyleSettingsCustomizable.SpacingOption.SPACE_AFTER_COMMA.name)
+            }
+        }
+    }
+
+    override fun getIndentOptionsEditor(): IndentOptionsEditor? = SmartIndentOptionsEditor()
```

Comment:
If we decide not to provide indent settings in this PR (i.e. `org.toml.ide.formatter.settings.TomlCodeStyleSettingsProvider.TomlCodeStyleMainPanel#initTabs` initializes only space tab), it will be redudant